### PR TITLE
Add addFragment/addHash method

### PR DESCRIPTION
**Pull request by fishead**
This PR solve issue #338

### CODE REVIEW

File: src/URI.js [hunk 1]
```diff
@@ -780,6 +780,21 @@
       throw new TypeError('URI.addQuery() accepts an object, string as the name parameter');
     }
   };
+
+  URI.setQuery = function(data, name, value) {
```
Comment (rodneyrehm) [inline src/URI.js]: do you mind adding tests for this to test/test.js?

Comment (fishead) [reply src/URI.js]: Sure, sorry so the delay, I got really busy there days.

Comment (rodneyrehm) [reply src/URI.js]: no worries… I'm leaving for 2 week vacation in 1h, sorry for *that* delay :(

Comment (fishead) [reply src/URI.js]: Tests added, happy vacation!

**Comment by rodneyrehm** [comment]
thank you for your time and patience!

**Pull request merged**